charts/managed-identity-wallets/templates/deployment.yaml

```diff
@@ -134,83 +147,183 @@ spec:
             memory: 256Mi
         ports:
         - containerPort: 8086
-      - name: catenax-acapy
+      - name: catenax-endorser-acapy
```

Comment:
Please be aware that the official eclipse foundation project is called tractus-x